.npmrc

```diff
@@ -0,0 +1 @@
+registry = http://0.0.0.0:4873/
```

Comment:
What do you need this for?

Reply:
I'm sorry, it slipped out among so many things I was doing at the same time, I'll delete it now. I use it to manage my dependencies through a local proxy and not die trying between npm installations, after all this is Cuba.